test/unit/spec/signaling/v2/twilioconnectiontransport.js

```diff
@@ -182,7 +183,7 @@ describe('TwilioConnectionTransport', () => {
           assert.deepEqual(message.peer_connections, test.peerConnectionManager.getStates());
           assert.equal(message.token, test.accessToken);
           assert.equal(message.type, 'connect');
-          assert.equal(message.version, 2);
+          assert.equal(message.version, 3);
```

Comment:
It looks like the Ice request below is for version 1 (line 200). If possible all the messages should reflect version 3. I don't think it's mandatory for this PR though.